CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* Bump: sentry-android to 5.2.0 (#594)
```

Comment:
We've been using this format in the Unity SDK which is handy to easily link to see what changed between versions. Otherwise just by reading the changelog it's hard to tell if one should bother upgrading:

```suggestion
* Bump Sentry Android SDK to 5.2.0 ([#594](https://github.com/getsentry/sentry-dart/pull/594))
  - [changelog](https://github.com/getsentry/sentry-java/blob/5.2.0/CHANGELOG.md)
  - [diff](https://github.com/getsentry/sentry-java/compare/5.1.2...5.2.0)
```

Reply:
That makes so much sense.

Reply:
mm not a bad idea